### ISSUE TITLE
Import from AndOTP?

### ISSUE THREAD

**Issue by AmarOk1412** [FoxAuth/FoxAuth#35]
Hi,

I am currently using andotp as a 2FA app (https://github.com/andOTP/andOTP)

I see that FoxAuth is offering some import/export from other apps. So, I am wondering if importing from AndOTP will be planned, hopefully, one day.

Regards,

**Comment by Riajyuu** [comment]
@AmarOk1412 Hi,
Indeed we have that plan, at least one day.
But we have now only two active contributor but a trunk of work, this means it can be really slow.
Hope this answer your question.

**Comment by AmarOk1412** [comment]
Yeah I understand what being a contributor means, I have a todo list for the next years too.

Feel free to close this ticket if you prefer to not have a ticket open for months :)
